components/admin_console/admin_definition.jsx

```diff
@@ -4301,6 +4301,15 @@ const AdminDefinition = {
                         placeholder: t('admin.experimental.experimentalPrimaryTeam.example'),
                         placeholder_default: 'E.g.: "teamname"',
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'ExperimentalSettings.UseNewSAMLLibrary',
+                        label: t('admin.experimental.experimentalUseNewSAMLLibrary.title'),
+                        label_default: 'Use Improved SAML Library (Beta):',
+                        help_text: t('admin.experimental.experimentalUseNewSAMLLibrary.desc'),
+                        help_text_default: 'Enable a more efficient SAML Library. Warning: Not all providers have been tested. If you experience issues, please contact support: [https://about.mattermost.com/support/](!https://about.mattermost.com/supporter/)',
```

Comment:
Is `https://about.mattermost.com/supporter/` correct? Seems to 404 for me. Actually, looks like the default here doesn't match the i18n.